barf/arch/arm/armbase.py

```diff
@@ -263,7 +263,7 @@ def max_instruction_size(self):
         """
         instruction_size_map = {
             ARCH_ARM_MODE_ARM : 4,
-            ARCH_ARM_MODE_THUMB : 2,
+            ARCH_ARM_MODE_THUMB : 4,
```

Comment:
I am not sure why this change. As long as I know it the instruction size for thumb is 2 bytes.